.github/actions/integration-tests/run-ci-stage1

```diff
@@ -215,9 +274,19 @@ function post_run_cmd {
 }
 
 function remove_microk8s_images {
-    for image in $(microk8s ctr images list "name~=client-server" | grep -v REF | awk '{ print $1 }'); do
-        microk8s ctr images remove --sync ${image}
+    local rc=1
+
+    for image in $(ssh ${CI_ENDPOINT_USER}@${CI_ENDPOINT_HOST} microk8s ctr images list "name~=client-server" | grep -v REF | awk '{ print $1 }'); do
+        ssh ${CI_ENDPOINT_USER}@${CI_ENDPOINT_HOST} microk8s ctr images remove --sync ${image}
+        rc=$?
+
+        if [ ${rc} != 0 ]; then
+            echo "ERROR: Could not remove ${image} from microk8s registry"
+            break
```

Comment:
hmm so you can't delete one and give up ?

Reply:
well, its a CI test...if it can't delete one then something unexpected happened and the test fails